streams/src/main/java/org/apache/kafka/streams/state/internals/RecordConverter.java

```diff
@@ -23,9 +23,11 @@
 public interface RecordConverter {
     ConsumerRecord<byte[], byte[]> convert(final ConsumerRecord<byte[], byte[]> record);
 
-    @SuppressWarnings("deprecation")
-    static RecordConverter converter() {
-        return record -> {
+    final class RecordConverters {
```

Comment:
Why this nesting? This is confusing. In doubt, create own top-level class.

Reply:
ok, done!